### ISSUE TITLE
Fix tests for ignoreValues in value-no-vendor-prefix

### ISSUE THREAD

**Issue by hudochenkov** [stylelint/stylelint#3092]
Ref: https://github.com/stylelint/stylelint/pull/3089#issuecomment-354662389

Use correct CSS. Use `a` instead of `.foo`.

**Comment by nalinbhardwaj** [comment]
Similar error in [property-no-unknown](https://github.com/stylelint/stylelint/blob/f6fbad37e1ffa84aed4a996b3829b4275e7320c5/lib/rules/property-no-unknown/__tests__/index.js#L15)

**Comment by ntwb** [comment]
A quick search of the repo resulted in 1030 matches of `.foo` in 83 files....

**Comment by jeddy3** [comment]
I think we've used `foo`, `bar` and `baz` for the names of classes _when the rule specifically targets classes_ e.g. `selector-max-class`. We've used `a` for the selector when the rule isn't concerned with the selector e.g. `property-no-unknown`... it was a simple convention to signal to a reviewer that the selectors within each test aren't important.

I thought [we'd document this](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#example-patterns), but on closer inspection, we don't explicitly state using `a` when the selector isn't important. I believe it's worth sticking with this convention and we should probably document it?

**Comment by ntwb** [comment]
Agreed, let's get this documented and we can then perform an audit of sorts so we are consistent throughout.

-----

I'm wondering if we could use [CSSTree](https://github.com/csstree) to _validate_ our code examples, we'd only do it for what should be _valid code examples_ and skip examples where invalid code is explicitly used, such as `colr` in `property-no-unknown` etc.

**Comment by ntwb** [comment]
Previously #1818, where the readme examples were updated for consistency:

> There’s quite a few inconsistencies to what code is used in our example patterns e.g. in most places we use the `a` type selector when the rule isn't concerned the selector itself, but in some cases we use `.foo`. This PR standardise everything to `a`, leaving `.foo` reserved for when the selector itself is being checked.
>
> I’m finding these consistencies quite distracting when trying to quickly parse issues. Sometimes it takes a while to deconstruct what an example is trying to communicate. Hopefully keeping everything but the key thing consistent will help with this.
>
>It also has the benefit of consistency for users looking through all the docs.
